build/teamcity/internal/cockroach/build/ci/open-new-bazel-builder-image-pr.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+# This script expects the following env vars to be set:
+#   BRANCH [branch name to be created in fork]
+#   VERSION [new version to echo into build/.bazelbuilderversion]
+#   GH_TOKEN [github token]
+
+dir="$(dirname $(dirname $(dirname $(dirname $(dirname $(dirname "${0}"))))))"
+source "$dir/teamcity-common-support.sh"
+
+git_ssh() {
+  # $@ passes all arguments to this function to the command
+  GIT_SSH_COMMAND="ssh -i /tmp/.cockroach-teamcity-key" git "$@"
+}
+
+# Configure git.
+git config --global user.email "teamcity@cockroachlabs.com"
+git config --global user.name "cockroach-teamcity"
+configure_git_ssh_key
+mv .cockroach-teamcity-key /tmp/.cockroach-teamcity-key
```

Comment:
Why do you need to move the key under /tmp? I don't think it'll be cleaned up in this case.

Reply:
Moved it to the temp dir along with the clone